### PR TITLE
Integration tests for neo4j

### PR DESCRIPTION
**Pull request by kermakovich**

### CODE REVIEW

File: src/test/java/helper/BaseNeo4jContainer.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package helper;
+
+import org.testcontainers.containers.Neo4jContainer;
+import org.testcontainers.utility.MountableFile;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@SuppressWarnings("JTCOP.RuleAllTestsHaveProductionClass")
+public class BaseNeo4jContainer extends Neo4jContainer {
```
Comment (h1alexbel) [inline src/test/java/helper/BaseNeo4jContainer.java:10]: @kermakovich prefer composition over inheritance

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/test/java/integration/DancerRepositoryIT.java [hunk 2]
```diff
@@ -16,6 +17,7 @@
 @SpringBootTest
 @ContextConfiguration(classes = TestConfig.class)
 @ActiveProfiles("test")
+@DirtiesContext
```
Comment (h1alexbel) [inline src/test/java/integration/DancerRepositoryIT.java:20]: @kermakovich what is the purpose of usage DirtiesContext?

Comment (kermakovich) [reply src/test/java/integration/DancerRepositoryIT.java:20]: @h1alexbel I use `neo4jContainer::getBoltUrl` in @DynamicPropertySource and without @DirtiesContext only first test class runs  correctly, but the other classes fail.

Comment (h1alexbel) [reply src/test/java/integration/DancerRepositoryIT.java:20]: OK

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**